tools/Voron.Recovery/Recovery.cs

```diff
@@ -363,16 +357,29 @@ public RecoveryStatus Execute(CancellationToken ct)
                 revisionsWriter.WriteEndObject();
                 
 
-                logFile.WriteLine(
+                LogOperation(
                     $"Discovered a total of {_numberOfDocumentsRetrieved:#,#;00} documents within {sw.Elapsed.TotalSeconds::#,#.#;;00} seconds.");
-                logFile.WriteLine($"Discovered a total of {_attachmentsHashs.Count:#,#;00} attachments.");
-                logFile.WriteLine($"Discovered a total of {_numberOfFaultedPages::#,#;00} faulted pages.");
+                LogOperation($"Discovered a total of {_attachmentsHashs.Count:#,#;00} attachments.");
+                LogOperation($"Discovered a total of {_numberOfFaultedPages::#,#;00} faulted pages.");
             }
             if (_cancellationRequested)
                 return RecoveryStatus.CancellationRequested;
             return RecoveryStatus.Success;
         }
 
+        private void LogInfo(string msg)
```

Comment:
drop use of these methods, that isn't a good idea. 
You are paying the cost of building the log string even if you aren't logging.